modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-an-operator.adoc

```diff
@@ -13,31 +13,53 @@ link:https://docs.openshift.com/container-platform/latest/operators/olm-what-ope
 
 .Procedure
 
-When deploying {prod-short} using the Operator, configure the intended strategy by modifying the `spec.k8s.ingressStrategy` property of the CheCluster Custom Resource object YAML file.
+When deploying {prod-short} using the Operator, configure the intended strategy by modifying the `spec.server.serverExposureStrategy` and  optionally `spec.server.singleHostWorkspaceExposureType` properties of the CheCluster Custom Resource object YAML file.
 To activate changes done to CheCluster YAML file, do one of the following:
 
 * Create a new cluster by executing the `{prod-cli}` command with applying a patch. For example:
 +
 [subs="+quotes,+attributes"]
 ----
-$ {prod-cli} server:start ... --che-operator-cr-patch-yaml=patch.yaml
+$ {prod-cli} server:start  --installer=operator --platform=... --che-operator-cr-patch-yaml=patch.yaml
 ----
 
-patch.yaml should contain the following:
+patch.yaml should contain the following for `native` single-host exposure type:
 
 [source,yaml,subs="+quotes"]
 ----
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  name: eclipse-che
 spec:
-  k8s:
-    ingressStrategy: '__single-host__'
+  server:
+    serverExposureStrategy: '__single-host__'
 ----
 
 
+For `gateway` type, patch.yaml should look as follows:
+
+[source,yaml,subs="+quotes"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  name: eclipse-che
+spec:
+  server:
+    serverExposureStrategy: 'single-host'
+    singleHostWorkspaceExposureType: 'gateway'
+    customCheProperties:
```

Comment:
`customCheProperties` should not be here. It was only temporal fix until we synchronized config properties between che-server and che-operator.